.github/workflows/ci_cd.yml

```diff
@@ -38,17 +38,33 @@ jobs:
 
       - name: Install dependencies
         run: |
+          sudo apt update
+          sudo apt-get install -y build-essential zip pandoc texlive-latex-extra latexmk texlive-pstricks
```

Comment:
I just added here `texlive-pstricks`, which was required by the PDF builder, @Revathyvenugopal162.